packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -77,7 +77,8 @@ export function getContainerAlreadyRegisteredErrorText(containerKey: ContainerKe
 
 /**
  * Properties for configuring the Devtools.
- * @alpha
+ * @beta
+ * @input
```

Comment:
I thought `@sealed` was only for classes but we do use it on interfaces even if the official docs only mention classes. So I'll suggest we apply it to all the interfaces in this API since they're not intended for external implementation.
```suggestion
 * @input
 * @sealed
```